python/mxnet/lr_scheduler.py

```diff
@@ -136,3 +136,42 @@ def __call__(self, num_update):
             else:
                 return self.base_lr
         return self.base_lr
+
+class PolyScheduler(LRScheduler):
+    """ Reduce the learning rate by given a list of steps.
+
+    Calculate the new learning rate by::
+
+       base_lr * (1-nup/max_nup)^pwr
+       if nup < max_nup, 0 otherwise.
+
+    Parameters
+    ----------
+       num_update: current number of updates
+       max_update: maximum number of updates before the decay reaches 0.
+       base_lr:    base learning rate
+       pwr:   power of the decay term as a funtion of the current number of updates.
+
+    """
+
+    def __init__(self, num_update, max_update, base_lr=0.01, pwr=2):
+        super(PolyScheduler, self).__init__(base_lr)
+        assert isinstance(max_update, int)
+        if max_update < 1:
+            raise ValueError("maximum number of updates must be strictly positive")
+        self.base_lr_orig = self.base_lr
+        self.max_update = max_update
+        self.power = pwr
+        self.count = num_update
+        if num_update <= max_update:
```

Comment:
This this duplicate with line 173. I understand it is for resume training, but that should be handled in `__call__`, see the example in MultiFactorScheduler. Therefore, num_update is not necessary in `__init__`

Reply:
@zhreshold i removed the num_update duplicate line, can you please check.thanks.